.circleci/config.yml

```diff
@@ -101,7 +101,12 @@ run_oss_benchmark: &run_oss_benchmark
       name: Run OSS Benchmark
       command: |
         python benchmarks/oss.py --check_regression --world_size 4 --reference_speed 21.2 --reference_memory 4220 --reference_loss 0.63
-        python benchmarks/oss.py --gloo --optim_type oss
+
+run_oss_gloo: &run_oss_gloo
```

Comment:
What is the rationale for breaking this out?

Reply:
it's not really a benchmark, just a correctness test because gloo will assert on some operations where NCCL will not. The setup (world size) is not the same for speed's sake, so the numbers in the log were not comparable.

This way all the values in oss benchmark are actual comparisons, I was planning to add [this alternative implementation](https://github.com/facebookresearch/fairscale/tree/oss_sharded_experimental) to the mix at some point and the logs looked[ like this](https://app.circleci.com/pipelines/github/facebookresearch/fairscale/565/workflows/a87b1c8f-d4c1-4cd0-87f3-a704fe9e8d21/jobs/2128/parallel-runs/0/steps/0-110), hard for external people to see what's going on